src/pilot/assign_role.py

```diff
@@ -846,7 +846,30 @@ def assign_role(ip_mac_service_tag, node_uuid, role_index,
     if role_index.index:
         role = "node:{}-{}".format(flavor, role_index.index)
 
-    value = "{},{},boot_mode:uefi,boot_option:local".format(role, node.properties['capabilities'])
+    # capabilities will be available if in-band introspection or
+    # out-of-band inspection is performed before
+    # assign_role.py is executed.
+    # if else statement needed to run this script before out-of-band
```

Comment:
Let's replace "if else statement" with "This conditional handling is" for better readability